extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -252,36 +250,47 @@ public String getParentChannelAvatarUrl() throws ParsingException {
         final JsonArray ajaxJson = getJsonResponse(page.getUrl(), getExtractorLocalization());
 
         JsonObject sectionListContinuation = ajaxJson.getObject(1).getObject("response")
-                .getObject("continuationContents").getObject("gridContinuation");
+                .getArray("onResponseReceivedActions").getObject(0).getObject("appendContinuationItemsAction");
```

Comment:
is it safe to assume that the `appendContinuationItemsAction` object is always in the first array element?

Reply:
I don't know

Reply:
youtube-dl also assume it's the first element. https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/extractor/youtube.py#L2507
but if you wanna be sure the safest way is a simple for loop.